sql/src/test/resources/calcite/tests/window/no_grouping.sqlTest

```diff
@@ -12,7 +12,7 @@ expectedOperators:
   - type: "window"
     processor:
       type: "framedAgg"
-      frame: { peerType: "ROWS", lowUnbounded: true, lowOffset: 0, uppUnbounded: true, uppOffset: 0 }
+      frame: { type: "rows" }
```

Comment:
why is this not `{ type: unbounded }` similar to the one in `no_grouping2.sqlTest`

Reply:
the test was disabled with `failingTest` ; enabled it and updated this